### PR TITLE
Release/6.0.0-alpha.2

### PR DESCRIPTION
**Pull request by joshdholtz**
### New Features
* [CF-1119] - Default offer now uses best offer logic (longest free, best savings, base plan) (#754) via Josh Holtz (@joshdholtz)
* [CF-1147] Use GoogleProrationMode in purchases (#752) via beylmk (@beylmk)
* [CF-1148] Add proration mode (#751) via beylmk (@beylmk)
### Other Changes
* [CF-1120] - Update SDK docs for default offer logic (#762) via Josh Holtz (@joshdholtz)
* Clean up Google-related TODOs (#736) via beylmk (@beylmk)
* [CF-1135] Remove requirement for `StoreProduct` when purchasing a `PurchaseOption` (#739) via Josh Holtz (@joshdholtz)

[CF-1119]: https://revenuecats.atlassian.net/browse/CF-1119?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CF-1147]: https://revenuecats.atlassian.net/browse/CF-1147?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CF-1148]: https://revenuecats.atlassian.net/browse/CF-1148?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CF-1135]: https://revenuecats.atlassian.net/browse/CF-1135?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[CF-1120]: https://revenuecats.atlassian.net/browse/CF-1120?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by joshdholtz** [comment]
⚠️ This is using https://github.com/RevenueCat/fastlane-plugin-revenuecat_internal/pull/47 for generating this PR

File: CHANGELOG.latest.md [hunk 1]
```diff
@@ -1 +1,8 @@
-Initial v6.0.0 alpha release
+### New Features
+* [CF-1119] - Default offer now uses best offer logic (longest free, best savings, base plan) (#754) via Josh Holtz (@joshdholtz)
+* [CF-1147] Use GoogleProrationMode in purchases (#752) via beylmk (@beylmk)
+* [CF-1148] Add proration mode (#751) via beylmk (@beylmk)
+### Other Changes
+* [CF1120] - Update SDK docs for default offer logic (#762) via Josh Holtz (@joshdholtz)
+* Clean up Google-related TODOs (#736) via beylmk (@beylmk)
+* [CF-1135] Remove requirement for `StoreProduct` when purchasing a `PurchaseOption` (#739) via Josh Holtz (@joshdholtz)
```
Comment (beylmk) [inline CHANGELOG.latest.md:8]: NABD, but i'd call that one a new feature!

**Review by beylmk — APPROVED** [review]
🥳

File: CHANGELOG.latest.md (hunk 1 above)
Comment (joshdholtz) [reply CHANGELOG.latest.md:8]: Oh yeah! Looks like I should probably make sure I give PRs proper tags 🤷‍♂️

Comment (beylmk) [reply CHANGELOG.latest.md:8]: oooo i forgot these are auto-generated now 😻

**Pull request merged**